ddtrace/internal/telemetry/data/metrics.py

```diff
@@ -0,0 +1,62 @@
+import time
+from typing import Dict
+from typing import List
+from typing import Literal
+from typing import Optional
+from typing import Tuple
+
+from ...hostname import get_hostname
+
+
+class MetricType:
```

Comment:
I removed metrics from this PR but I will keep this in mind

Reply:
This change was made [here](https://github.com/DataDog/dd-trace-py/blob/b095f01dfbd61c53f72c3ff8b0e8d2d9a146db48/ddtrace/internal/telemetry/metrics.py#L12) and it can be found in this PR: Python Telemetry Metrics #3071